javascripting/array-filtering.js

```diff
@@ -0,0 +1,7 @@
+var numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+var filtered = numbers.filter(function(number) {
+  return number % 2 === 0;
```

Comment:
I'm sticking with Douglas Crockford on this one. Just because you can doesn't necessarily mean you should. I never use `==` in JavaScript, and in fact JS linters will complain if you do. http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons